### PR TITLE
SCALRCORE-30839 Provider > don't read `secret-key` webhook attribute on read operations

### PR DESCRIPTION
**Pull request by RomanMytsko**
### Changelog
* [x] I have updated CHANGELOG.md according to [CONTRIBUTING.md](../CONTRIBUTING.md)

### Documentation
* [ ] I have updated resource/data source documentation in [docs](../docs)

### State
* [ ] My changes affect the state
* [ ] I have included a state migration and a unit test for it

### CODE REVIEW

File: scalr/resource_scalr_webhook.go [hunk 1]
```diff
@@ -433,7 +433,6 @@ func readNewWebhook(ctx context.Context, d *schema.ResourceData, scalrClient *sc
 	_ = d.Set("enabled", webhook.Enabled)
 	_ = d.Set("last_triggered_at", webhook.LastTriggeredAt)
 	_ = d.Set("url", webhook.Url)
-	_ = d.Set("secret_key", webhook.SecretKey)
```
Comment (RomanMytsko) [inline scalr/resource_scalr_webhook.go:436]: Даний атрибут будемо [робити](https://scalr-labs.atlassian.net/browse/SCALRCORE-30859) `sensitive`, тому щоб уникнути дріфтів потрібно перестати читати це поле.

Comment (petroprotsakh) [reply scalr/resource_scalr_webhook.go:436]: nit: please use English in public repositories

Comment (petroprotsakh) [inline scalr/resource_scalr_webhook.go:436]: What about the `scalr_webhook` datasource?

Comment (RomanMytsko) [reply scalr/resource_scalr_webhook.go:436]: @petroprotsakh Updated the AC in the ticket. This attribute will be removed from the data source.

**Review by petroprotsakh — APPROVED** [review]

**Pull request merged**